lib/reader.js

```diff
@@ -14,6 +19,10 @@ Reader.OPEN_FILE = 1;
 Reader.OPEN_URI = 2;
 Reader.OPEN_LOCAL = 3;
 
+if(typeof require === 'function') {
```

Comment:
duplicated from above, this one can be removed it seems

Reply:
This one is required, because `fs` needs to be required in when running in node environments. the other one is more about error messages to the user if open local is attempted in a browser.